client/main.lua

```diff
@@ -1,4 +1,6 @@
 -- Variables
+local config = require 'config.client'
+local sharedConfig = require 'config.shared'
```

Comment:
You can do something like require 'config.shared.price' to get a specific table value. This might be preferable to limit the scope instead of having the variable be the entire table